core/src/components/picker-column/picker-column.tsx

```diff
@@ -90,8 +90,8 @@ export class PickerColumnCmp implements ComponentInterface {
   }
 
   disconnectedCallback() {
-    cancelAnimationFrame(this.rafId);
-    clearTimeout(this.tmrId);
+    if (this.rafId !== undefined) cancelAnimationFrame(this.rafId);
+    if (this.tmrId) clearTimeout(this.tmrId);
```

Comment:
Checking for `undefined` explicitly isn't necessary because `tmrId` is of type `NodeJS.Timeout | undefined`. Compare with `rafId` which is actually type `number | undefined`, so checking for `undefined` explicitly avoids tripping over the case where the value is 0.

Reply:
Oh interesting. It looks like we can assume that `requestAnimationFrame` returns a non-zero value: https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame#return_value And then `setTimeout` returns a positive integer: https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value

Reply:
ESLint complains without the explicit check (since it's trying to help us avoid that 0 case) so I'll keep it to avoid needing to disable the rule or something.